test/e2e/test-apps/other/scope-breadcrumbs/src/index.html

```diff
@@ -9,6 +9,10 @@
 
       init({
         debug: true,
+        beforeSend: (event, hint) => {
+          hint.attachments = [{ filename: 'screenshot.png', data: 'captureScreen()' }, ...(hint.attachments || [])];
```

Comment:
I assume this is just an extra assertion you snuck into this PR? :D

Reply:
Before you could add attachments from the renderer process by adding them to the scope which gets synced to the main process. 

This PR fixes the ability to add envelopes via `beforeSend` too.

So this extra check is to ensure that envelopes from both routes make it into the final envelope with neither getting overwritten by the other or any other shenanigans.